package.json

```diff
@@ -206,6 +206,7 @@
     "@web/test-runner-mocha": "^0.7.5",
     "@web/test-runner-playwright": "^0.9.0",
     "@web/test-runner-saucelabs": "^0.8.0",
+    "chai": "^4.3.7",
```

Comment:
why are we adding `chai`? Does it pull along types? There's also `@types/chai`

Reply:
Was trying to fix a lint error but ended up going with another approach, removed

Reply:
Was trying to fix a lint error but ended up going with another approach, removed